controllers/storagecluster/cephcluster.go

```diff
@@ -1335,6 +1335,6 @@ func determineDefaultCephDeviceClass(foundDeviceClasses []rookCephv1.DeviceClass
 			}
 		}
 	}
-	// By default return "ssd"
-	return "ssd"
+	// if no device classes are found in status return empty string
+	return ""
```

Comment:
Looks good to not have a default unless one has been found on the CephCluster CR. As far as past impact of this `ssd` default, what was the behavior when the user set a deviceClass explicitly in the StorageCluster CR? Were we setting this default `ssd` incorrectly? Just wondering if we had a bug there.

Reply:
Yeah, it was returning this value by default, but as soon as a deviceclass would be found on the cephcluster CR status, it would change to the actual deviceclass. So I don't think it was a bug but yeah it was misleading though.